### PR TITLE
Enable installer tests in our CI.

### PR DESCRIPTION
**Pull request by eerhardt**
@Sridhar-MS @piotrpMSFT @mmitche

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -28,6 +28,9 @@ def static getBuildJobName(def configuration, def os) {
             if (os == 'Windows_NT') {
                 buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} Default"
             }
+            else if (os == 'Windows_2016') {
+                buildCommand = ".\\build.cmd -Configuration ${lowerConfiguration} -RunInstallerTestsInDocker Default"
+            }
```
Comment (Sridhar-MS) [inline netci.groovy]: This is going to make some CI to run on `Windows_NT` and some on `Windows_2016`. And the MSI tests will not be deterministic-ally run every time. 

We can do all the CI runs only on `Windows_2016`.

Comment (eerhardt) [inline netci.groovy]: My plan was to turn on Windows 2016 and keep running Windows NT at the same time. Then when Windows 2016 is stable, remove Windows NT.

Does that make sense?

**Comment by piotrpMSFT** [comment]
Let's talk about this tomorrow. Platform matrix is shaping up. We need to light up a lot of new CI!

**Comment by eerhardt** [comment]
@dotnet-bot test Windows_NT Debug Build
@dotnet-bot test Windows_NT Release Build

**Pull request merged**